runbotics/runbotics-scheduler/src/scheduler-database/credential/credential.service.ts

```diff
@@ -222,13 +230,17 @@ export class CredentialService {
       tenantId: credential.tenantId,
     });
 
+    const credentialOldName =
+        credential.name !== credentialDto.name ? credential.name : undefined;
```

Comment:
this should be adjusted depending on what approach you're gonna take in the interfaces